src/ui/liferea_shell.c

```diff
@@ -236,14 +237,15 @@ liferea_shell_restore_position (void)
 
 	/* Restore position only if the width and height were saved */
 	if (w != 0 && h != 0) {
+		gdk_monitor_get_workarea (gdk_display_get_primary_monitor (gdk_display_get_default ()), &work_area);
```

Comment:
It could be better to try to get the monitor that contains the window, with gdk_display_get_monitor_at_window.

Reply:
for sure

Reply:
@ghost Do you have time to work on this?

Reply:
@lwindolf It looks like the pull request author deleted their account. But the next commit 6989728 fixes that issue. Sorry I didn't mark it as resolved earlier.
It may be nitpicking but the display could also be obtained with `gtk_widget_get_display` instead of `gdk_display_get_default`